static_src/components/usage_and_limits.jsx

```diff
@@ -135,6 +255,13 @@ export default class UsageAndLimits extends React.Component {
             </PanelRow>
           </PanelGroup>
         </PanelGroup>
+        <PanelGroup>
+          <PanelRow>
+            <div className={ this.styler('panel-controls') }>
```

Comment:
I actually added a class that does something similar to this, but will update once my stuff gets merged.